lib/hanami/logger.rb

```diff
@@ -103,6 +103,10 @@ class Formatter < ::Logger::Formatter
       # @api private
       SEPARATOR = ' '.freeze
 
+      # @since x.x.x
+      # @api private
+      NEW_LINE = "\n".freeze
```

Comment:
FYI, this won't work on Windows. You can use the special EOL constant `$/` which works everywhere